### DISCUSSION TITLE
EM model verification (basic 50R trace) and XY plot of impedance vs frequency

### DISCUSSION THREAD

**Discussion by oberstet** [jankae/LibreVNA#251] [Q&A]
Hi all,

I would like to calibrate my openEMS models with LibreVNA.

This is quite a challenge, as I am a newbie;)

Currently, I am stuck - any hints or tips would be greatly appreciated!

Thanks a lot!!

/Tobias

PS: fwiw, I also posted this Q in expanded form here https://www.edaboard.com/threads/verification-of-openems-modeling-with-real-world-using-librevna.410800/

----

My idea was to *verify the PCB trace width my EM model _claims_ to match for 50R*  with the PCB stack I modeled (2-layer JLCPCB Teflon/PTFE) in openEMS vs the real-world PCB tiles measured with LibreVNA.

This is what I have:

* PCB tiles with a single PCB traces of different widths (1.3mm - 2.4mm)
* 2 PCB test fixtures with currently 1.9mm and 1.3mm PCB traces installed
* LibreVNA, calibrated using the included calkit

My EM model tells me 1.9mm is the best match to 50R (which is what I want to verify), and I have measured S11 using LibreVNA (pls see below) for

1. 1.9mm
2. 1.3mm
3. THRU included with LibreVNA

It "looks like to me" that of those 3, indeed both 1. and 3. look "close to 50R", and 2. does not - that would match what my EM model predicts.

But I'm not sure.

Ideally, I would like to measure complete XY Plots of impedance vs frequency and have a look in detail (rather than looking at the Smith chart).

Even better, I would like a single number summarizing "fit to 50R" over whole frequency range (300M-3G) that I set.

*How do I approach EM model verification starting ground-up? Given the pieces I have?*

My model claims 1.9mm would be 50R, and it does "look like" as this isn't totally off, but how do I do that systematically?
Thank you very much!!

-----

The pieces I used, including PCB tiles and test fixtures

![PXL_20240426_112328696](https://github.com/jankae/LibreVNA/assets/233340/1bfe9c77-0091-4a2e-91e5-b996cf140d51)

-----

## LibreVNA THRU

![CAL-THRU-Screenshot from 2024-04-26 13-18-26](https://github.com/jankae/LibreVNA/assets/233340/a3962617-4243-41d5-a1ac-89065a5858af)

## PCB Trace 1.9mm

![TRACE-1 9mm-Screenshot from 2024-04-26 13-15-06](https://github.com/jankae/LibreVNA/assets/233340/5df1cfe0-8841-416f-8193-0f6f11ddd6ae)

## PCB Trace 1.3mm

![TRACE-1 3mm-Screenshot from 2024-04-26 13-13-34](https://github.com/jankae/LibreVNA/assets/233340/3dc72b0e-d79f-4e07-a294-fb7743f2617c)

**Comment by jankae** [comment]
You could try the TDR feature, it should show you the impedance of your trace vs. distance (instead of S11 vs. frequency). Ideally, this will show up as (close to) 50 ohms for the 1.9mm trace and as something further away from 50 ohms for the 1.3mm trace.

**Comment by dominikee** [comment]
Hey sorry to necro this thread, I was just curious about the housings and connectors that you (@oberstet) used in your picture. Any chance you have a link to them or some info about where to find them? They could be really useful for my project. Thanks for any guidance!
